Client/Info.plist

```diff
@@ -76,25 +68,43 @@
 	</array>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>LSSupportsOpeningDocumentsInPlace</key>
+	<true/>
+	<key>LeanplumAppId</key>
+	<string></string>
+	<key>LeanplumDevelopmentKey</key>
+	<string></string>
+	<key>LeanplumProductionKey</key>
+	<string></string>
+	<key>MozDevelopmentTeam</key>
+	<string>$(DEVELOPMENT_TEAM)</string>
+	<key>MozWhatsNewTopic</key>
+	<string>whats-new-ios-18</string>
 	<key>NSAppTransportSecurity</key>
 	<dict>
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
-	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Websites you visit may request your location.</string>
 	<key>NSCameraUsageDescription</key>
 	<string>This lets you take and upload photos.</string>
+	<key>NSFaceIDUsageDescription</key>
+	<string>Firefox requires Face ID to access your saved logins.</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>Websites you visit may request your location.</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>This lets you take and upload videos.</string>
 	<key>NSPhotoLibraryAddUsageDescription</key>
 	<string>This lets you save photos.</string>
-	<key>NSFaceIDUsageDescription</key>
-	<string>Firefox requires Face ID to access your saved logins.</string>
 	<key>NSUserActivityTypes</key>
 	<array>
+		<string>QuickActionIntent</string>
+		<string>QuickLinkSelectionIntent</string>
```

Comment:
Is this the right place to list these intents? I just came across this because I was looking into better Hand-off support between Firefox for iOS and Firefox Desktop. The NSUserActivityTypes documentation says that these types should have reverse-DNS syntax, e.g. `org.mozilla.ios.firefox.quick-action`.